network-store-server/src/test/java/com/powsybl/network/store/server/ExtensionHandlerTest.java

```diff
@@ -289,4 +290,38 @@ public void updateExtensionsFromEquipmentsTest() {
         assertEquals(10.0, activePowerControl.getDroop(), 0.1);
         assertEquals(2.0, activePowerControl.getParticipationFactor(), 0.1);
     }
+
+    @Test
+    public void insertNonPersistentExtensionTest() {
+        String equipmentIdA = "idBatteryA";
+
+        OwnerInfo infoBatteryA = new OwnerInfo(
```

Comment:
Absolutely unimportant but I find it strange to call it sometimes equipmentA and sometimes BatteryA.

Actually even the "A" is suspicisous, when there is only one equipment. It looks like a copy paste!

Reply:
It was copied from another test